src/main/java/com/adyen/model/checkout/CreatePaymentLinkRequest.java

```diff
@@ -50,6 +66,10 @@
     @SerializedName("countryCode")
     private String countryCode = null;
 
+    @SerializedName("deliverAt")
+    @JsonAdapter(DateTimeGMTSerializer.class)
+    private Date deliverAt = null;
```

Comment:
we have `expiresAt` as a String, probably `deliverAt` is autogenerated to Date because of the "format" provided on the specs side. Since "type" is still String on the specs, can you double check with e2e if this field works as expected?

Reply:
I internally in the core also it is as Date, i have checked with e2e and it works fine.